pytorch_lightning/loggers/wandb.py

```diff
@@ -254,7 +254,6 @@ def __init__(
         offline: Optional[bool] = False,
         id: Optional[str] = None,
         anonymous: Optional[bool] = None,
-        version: Optional[str] = None,
```

Comment:
If this is removed without deprecation, it should be mentioned in the "Removed" section of the CHANGELOG

Reply:
shouldn't this be deprecated first?

Reply:
@rohitgr7 it's a very weird bug. Keeping `version=None` and providing a deprecation warning doesn't work. This works correctly only if the `version` argument is removed. So I think I can add it to the "Removed" section of the CHANGELOG as suggested by @carmocca .